feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailScreen.kt

```diff
@@ -39,68 +50,103 @@ fun MovieDetailScreen(
   }
 
   if (state.isLoading) {
-    CircularProgressIndicator()
+    CircularProgressIndicator(
+      modifier = Modifier.wrapContentSize(),
+    )
   } else if (state.message != null) {
     Text(text = state.message!!)
   } else if (videoDetail != null) {
-    Column(
+    MovieDetailContent(
+      videoDetail = videoDetail,
+      state = state,
+      onBackPressed = onBackPressed,
+      onPlayPressed = viewModel::loadStreamInfo,
+    )
+  }
+}
+
+@Composable
+fun MovieDetailContent(
+  videoDetail: VideoDetail,
+  state: MovieDetailState,
+  onBackPressed: () -> Unit,
+  onPlayPressed: () -> Unit,
+) {
+  Column(
+    modifier = Modifier.verticalScroll(rememberScrollState()),
+    verticalArrangement = Arrangement.spacedBy(16.dp),
+  ) {
+    Box(
       modifier = Modifier
-        .verticalScroll(rememberScrollState()),
-      verticalArrangement = Arrangement.spacedBy(16.dp),
+        .height(300.dp),
     ) {
       AsyncImage(
         modifier = Modifier
           .fillMaxWidth()
           .aspectRatio(16f / 9f),
-        contentScale = ContentScale.Crop,
+        contentScale = ContentScale.FillWidth,
```

Comment:
The only issue with `ContentScale.FillWidth` is that it does not look good on phone. But we can fix it later.

<img src="https://github.com/moallemi/Film-Time/assets/1250691/9e4f5930-48b2-4a0e-a292-5ac78f635e54" width="50%">